### ISSUE TITLE
Setting locations not work

### ISSUE THREAD

**Issue by andrei923** [Sashie/skript-yaml#17]
Setting locations not work, with the new version of skript(2.3-beta1)

```
command test:
	trigger:
		load yaml "plugins/test.yml" as "test"
		set yaml value "loc" from "test" to location of player
		save yaml "test"
```

```
[21:59:24 ERROR]: Could not pass event PlayerCommandPreprocessEvent to Skript v2.3-beta1
org.bukkit.event.EventException: null
        at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:308) ~[spigot.jar:git-Spigot-a1ba5fe-b4230a9]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[spigot.jar:git-Spigot-a1ba5fe-b4230a9]
        at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:500) ~[spigot.jar:git-Spigot-a1ba5fe-b4230a9]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:485) ~[spigot.jar:git-Spigot-a1ba5fe-b4230a9]
        at net.minecraft.server.v1_13_R2.PlayerConnection.handleCommand(PlayerConnection.java:1638) ~[spigot.jar:git-Spigot-a1ba5fe-b4230a9]
        at net.minecraft.server.v1_13_R2.PlayerConnection.a(PlayerConnection.java:1481) ~[spigot.jar:git-Spigot-a1ba5fe-b4230a9]
        at net.minecraft.server.v1_13_R2.PacketPlayInChat.a(PacketPlayInChat.java:45) ~[spigot.jar:git-Spigot-a1ba5fe-b4230a9]
        at net.minecraft.server.v1_13_R2.PacketPlayInChat.a(PacketPlayInChat.java:1) ~[spigot.jar:git-Spigot-a1ba5fe-b4230a9]
        at net.minecraft.server.v1_13_R2.PlayerConnectionUtils.a(SourceFile:10) ~[spigot.jar:git-Spigot-a1ba5fe-b4230a9]
        at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_171]
        at java.util.concurrent.FutureTask.run(Unknown Source) [?:1.8.0_171]
        at net.minecraft.server.v1_13_R2.SystemUtils.a(SourceFile:199) [spigot.jar:git-Spigot-a1ba5fe-b4230a9]
        at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:900) [spigot.jar:git-Spigot-a1ba5fe-b4230a9]
        at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:417) [spigot.jar:git-Spigot-a1ba5fe-b4230a9]
        at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:835) [spigot.jar:git-Spigot-a1ba5fe-b4230a9]
        at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:733) [spigot.jar:git-Spigot-a1ba5fe-b4230a9]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_171]
Caused by: java.lang.NoSuchMethodError: me.sashie.skriptyaml.utils.yaml.SkriptYamlRepresenter.representMapping(Lorg/yaml/snakeyaml/nodes/Tag;Ljava/util/Map;Ljava/lang/Boolean;)Lorg/yaml/snakeyaml/nodes/Node;
        at me.sashie.skriptyaml.utils.yaml.SkriptYamlRepresenter.access$4(SkriptYamlRepresenter.java:1) ~[?:?]
        at me.sashie.skriptyaml.utils.yaml.SkriptYamlRepresenter$RepresentLocation.representData(SkriptYamlRepresenter.java:106) ~[?:?]
        at org.yaml.snakeyaml.representer.BaseRepresenter.representData(BaseRepresenter.java:89) ~[spigot.jar:git-Spigot-a1ba5fe-b4230a9]
        at org.yaml.snakeyaml.representer.BaseRepresenter.representMapping(BaseRepresenter.java:157) ~[spigot.jar:git-Spigot-a1ba5fe-b4230a9]
        at org.yaml.snakeyaml.representer.SafeRepresenter$RepresentMap.representData(SafeRepresenter.java:315) ~[spigot.jar:git-Spigot-a1ba5fe-b4230a9]
        at org.yaml.snakeyaml.representer.BaseRepresenter.representData(BaseRepresenter.java:95) ~[spigot.jar:git-Spigot-a1ba5fe-b4230a9]
        at org.yaml.snakeyaml.representer.BaseRepresenter.represent(BaseRepresenter.java:65) ~[spigot.jar:git-Spigot-a1ba5fe-b4230a9]
        at org.yaml.snakeyaml.Yaml.dumpAll(Yaml.java:306) ~[spigot.jar:git-Spigot-a1ba5fe-b4230a9]
        at org.yaml.snakeyaml.Yaml.dump(Yaml.java:285) ~[spigot.jar:git-Spigot-a1ba5fe-b4230a9]
        at me.sashie.skriptyaml.utils.yaml.YAMLProcessor.save(YAMLProcessor.java:316) ~[?:?]
        at me.sashie.skriptyaml.skript.EffSaveYaml.execute(EffSaveYaml.java:45) ~[?:?]
        at ch.njol.skript.lang.Effect.run(Effect.java:52) ~[?:?]
        at ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:61) ~[?:?]
        at ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:89) ~[?:?]
        at ch.njol.skript.lang.Trigger.execute(Trigger.java:57) ~[?:?]
        at ch.njol.skript.command.ScriptCommand.execute2(ScriptCommand.java:285) ~[?:?]
        at ch.njol.skript.command.ScriptCommand.execute(ScriptCommand.java:242) ~[?:?]
        at ch.njol.skript.command.Commands.handleCommand(Commands.java:257) ~[?:?]
        at ch.njol.skript.command.Commands$1.onPlayerCommand(Commands.java:172) ~[?:?]
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_171]
        at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_171]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_171]
        at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_171]
        at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:304) ~[spigot.jar:git-Spigot-a1ba5fe-b4230a9]
        ... 16 more
[21:59:24 ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing command 'test' in plugin Skript v2.3-beta1
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:46) ~[spigot.jar:git-Spigot-a1ba5fe-b4230a9]
        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:141) ~[spigot.jar:git-Spigot-a1ba5fe-b4230a9]
        at org.bukkit.craftbukkit.v1_13_R2.CraftServer.dispatchCommand(CraftServer.java:699) ~[spigot.jar:git-Spigot-a1ba5fe-b4230a9]
        at net.minecraft.server.v1_13_R2.PlayerConnection.handleCommand(PlayerConnection.java:1646) ~[spigot.jar:git-Spigot-a1ba5fe-b4230a9]
        at net.minecraft.server.v1_13_R2.PlayerConnection.a(PlayerConnection.java:1481) ~[spigot.jar:git-Spigot-a1ba5fe-b4230a9]
        at net.minecraft.server.v1_13_R2.PacketPlayInChat.a(PacketPlayInChat.java:45) ~[spigot.jar:git-Spigot-a1ba5fe-b4230a9]
        at net.minecraft.server.v1_13_R2.PacketPlayInChat.a(PacketPlayInChat.java:1) ~[spigot.jar:git-Spigot-a1ba5fe-b4230a9]
        at net.minecraft.server.v1_13_R2.PlayerConnectionUtils.a(SourceFile:10) ~[spigot.jar:git-Spigot-a1ba5fe-b4230a9]
        at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_171]
        at java.util.concurrent.FutureTask.run(Unknown Source) [?:1.8.0_171]
        at net.minecraft.server.v1_13_R2.SystemUtils.a(SourceFile:199) [spigot.jar:git-Spigot-a1ba5fe-b4230a9]
        at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:900) [spigot.jar:git-Spigot-a1ba5fe-b4230a9]
        at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:417) [spigot.jar:git-Spigot-a1ba5fe-b4230a9]
        at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:835) [spigot.jar:git-Spigot-a1ba5fe-b4230a9]
        at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:733) [spigot.jar:git-Spigot-a1ba5fe-b4230a9]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_171]
Caused by: java.lang.NoSuchMethodError: me.sashie.skriptyaml.utils.yaml.SkriptYamlRepresenter.representMapping(Lorg/yaml/snakeyaml/nodes/Tag;Ljava/util/Map;Ljava/lang/Boolean;)Lorg/yaml/snakeyaml/nodes/Node;
        at me.sashie.skriptyaml.utils.yaml.SkriptYamlRepresenter.access$4(SkriptYamlRepresenter.java:1) ~[?:?]
        at me.sashie.skriptyaml.utils.yaml.SkriptYamlRepresenter$RepresentLocation.representData(SkriptYamlRepresenter.java:106) ~[?:?]
        at org.yaml.snakeyaml.representer.BaseRepresenter.representData(BaseRepresenter.java:89) ~[spigot.jar:git-Spigot-a1ba5fe-b4230a9]
        at org.yaml.snakeyaml.representer.BaseRepresenter.representMapping(BaseRepresenter.java:157) ~[spigot.jar:git-Spigot-a1ba5fe-b4230a9]
        at org.yaml.snakeyaml.representer.SafeRepresenter$RepresentMap.representData(SafeRepresenter.java:315) ~[spigot.jar:git-Spigot-a1ba5fe-b4230a9]
        at org.yaml.snakeyaml.representer.BaseRepresenter.representData(BaseRepresenter.java:95) ~[spigot.jar:git-Spigot-a1ba5fe-b4230a9]
        at org.yaml.snakeyaml.representer.BaseRepresenter.represent(BaseRepresenter.java:65) ~[spigot.jar:git-Spigot-a1ba5fe-b4230a9]
        at org.yaml.snakeyaml.Yaml.dumpAll(Yaml.java:306) ~[spigot.jar:git-Spigot-a1ba5fe-b4230a9]
        at org.yaml.snakeyaml.Yaml.dump(Yaml.java:285) ~[spigot.jar:git-Spigot-a1ba5fe-b4230a9]
        at me.sashie.skriptyaml.utils.yaml.YAMLProcessor.save(YAMLProcessor.java:316) ~[?:?]
        at me.sashie.skriptyaml.skript.EffSaveYaml.execute(EffSaveYaml.java:45) ~[?:?]
        at ch.njol.skript.lang.Effect.run(Effect.java:52) ~[?:?]
        at ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:61) ~[?:?]
        at ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:89) ~[?:?]
        at ch.njol.skript.lang.Trigger.execute(Trigger.java:57) ~[?:?]
        at ch.njol.skript.command.ScriptCommand.execute2(ScriptCommand.java:285) ~[?:?]
        at ch.njol.skript.command.ScriptCommand.execute(ScriptCommand.java:242) ~[?:?]
        at ch.njol.skript.command.ScriptCommand.onCommand(ScriptCommand.java:193) ~[?:?]
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:44) ~[spigot.jar:git-Spigot-a1ba5fe-b4230a9]
        ... 15 more
```

**Comment by UzumakiYT** [comment]
that's the latest release

**Comment by AlexLew95** [comment]
oh yes my bad, I meant use skript dev37c and not alpha or beta versions

**Comment by Sashie** [comment]
fixed in next version, 1.13 has a different version of snakeyaml
